res/res_pjsip/config_transport.c

```diff
@@ -1246,6 +1311,34 @@ static int transport_tls_method_handler(const struct aco_option *opt, struct ast
 	return 0;
 }
 
+static int transport_tcp_keepalive_bool_handler(const struct aco_option *opt, struct ast_variable *var, void *obj) {
+    struct ast_sip_transport *transport = obj;
+
+    if (!strcasecmp(var->name, "tcp_keepalive_enable")) {
+        transport->tcp_keepalive_enable = ast_true(var->value) ? 1 : 0;
+    } else {
+        return -1;
+    }
+
+    return 0;
+}
+
+static int transport_tcp_keepalive_int_handler(const struct aco_option *opt, struct ast_variable *var, void *obj) {
```

Comment:
This does not need to be implemented. The configuration framework provides either an OPT_UINT_T or OPT_INT_T which implements things in a more complete manner and adds error checking.